merlin/models/tf/inputs/embedding.py

```diff
@@ -399,25 +402,49 @@ def Embeddings(
     ParallelBlock
         Returns a parallel block with an embedding table for each categorical features
     """
+    if sequence_combiner and "combiner" in kwargs:
+        raise ValueError(
+            "`combiner` cannot be specified as a keyword argument ",
+            "when `sequence_combiner` is specified",
+        )
+
+    if trainable:
+        kwargs["trainable"] = trainable
+    if embeddings_regularizer:
+        kwargs["embeddings_initializer"] = embeddings_initializer
+    if embeddings_regularizer:
+        kwargs["embeddings_regularizer"] = embeddings_regularizer
+    if activity_regularizer:
+        kwargs["activity_regularizer"] = activity_regularizer
 
     tables = {}
-    trainable = trainable or {}
 
     for col in schema:
-        tables[col.name] = EmbeddingTable(
-            _get_dim(col, embedding_dims, infer_dim_fn),
-            col,
-            combiner=_get_combiner(col, sequence_combiner),
-            embeddings_initializer=_get_initializer(col, embeddings_initializers),
-            trainable=trainable.get(col.name, True),
-            **kwargs,
-        )
+        table_kwargs = _forward_kwargs_to_table(col, table_cls, kwargs)
+        if sequence_combiner:
+            table_kwargs["combiner"] = _get_combiner(col, sequence_combiner)
```

Comment:
It's not clear to my why this param requires a different name in this function. If `sequence_combiner` is preferred over `combiner` it seems like that should be the name in the `EmbeddingTable` too. Or we change this to match `combiner` from the EmbeddingTable?